addons/account/views/account_move_views.xml

```diff
@@ -570,9 +570,11 @@
                         <!-- Post -->
                         <button name="action_post" string="Post" class="oe_highlight"
                                 type="object" groups="account.group_account_invoice" data-hotkey="v"
+                                context="{'validate_analytic': True}"
```

Comment:
Can we add the context to the list view as well for the multi-edit?
Or can it be propagated when we don't want it?

Reply:
> Can we add the context to the list view as well for the multi-edit?
Since there are multiple lines (with different products and accounts), the list view (multi-edit) has a forced applicability to optional - so it would never be invalid.

>Or can it be propagated when we don't want it?
Since it is on a button - it should not be propagated. But if we added the context key to a `window action`, or the button returns one, then it might be. I'm not sure how we could prevent that?